### ISSUE TITLE
How to action failed and/or skipped files.

### ISSUE THREAD

**Issue by Ritzo90** [MicrosoftDocs/OfficeDocs-SharePoint#3458]
Hi, 
this is a great free tool so thank you. The issue we have had is we expected ~100 GB of files and the scan only comes back with 60 GB. This was due partly due to the failed files/shortcuts that we had to fix. It would be nice if they were copied and the shortcut removed like we did manually.

More of an issue is the 40+ GB of files seemingly skipped with no reason provided. Please add information on navigating skipped files.

I've raised a case for support, most forums suggest using a third party tool instead.

Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 35884f2c-5c45-1246-16cc-48b47217a07c
* Version Independent ID: e5c10a91-82b4-ade5-34ed-9bcc349564ef
* Content: [Step 2: Scan and assess Google accounts using Migration Manager - Migrate to Microsoft 365](https://learn.microsoft.com/en-us/sharepointmigration/mm-google-step2-scan-assess)
* Content Source: [migration/mm-google-step2-scan-assess.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/public/migration/mm-google-step2-scan-assess.md)
* Service: **sharepoint-online**
* Sub-service: **sharepoint-migration**
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

**Comment by scanum** [comment]
Thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.

**Comment by Ritzo90** [comment]
Hi,
We were unable to pinpoint the failed files and the service rep suggested the following in case 33237048:
_"I was able to check with the team however we could not find any information about the failed files as it is not getting reflected in the logs.

**Usually this are some of the reasons why file gets skipped or failed:**
Google doesn't allow us to export Drawings, Forms, Sites, and Maps from Drive. These aren't migrated.
Shortcut files aren't supported for migration and therefore not migrated.
Files shared with the user aren't migrated"_

We closed the case as it's likely one of the above and 36.2GB was successfully migrated.
